### ISSUE TITLE
info addon

### ISSUE THREAD

**Issue by PierreTurnbull** [storybooks/storybook#4559]
I would just like to let you know that I find the [info addon](https://github.com/storybooks/storybook/tree/master/addons/info) documentation hard to follow. I am struggling to install it because it lacks informations. For example, I am requested to use `addDecorator(withInfo)` in the config file but I don't know where, how to import it (if needed) and same for `withInfo`. This doc could use some update!

**Comment by jlpospisil** [comment]
I wasn't able to get the global decorator to work like that either.  The following worked for me (obviously the options could be changed):

addDecorator((getStory, context) => withInfo({ inline: true })(getStory)(context));

**Comment by nickstaroba** [comment]
I found updated documentation here: 

https://github.com/storybooks/storybook/tree/master/addons/info

**Comment by PierreTurnbull** [comment]
@nickstaroba you wrote the exact same link as I did...

**Comment by vogelino** [comment]
I must agree with @PierreTurnbull, the documentation should include the following import:
```js
import { addDecorator } from '@storybook/react';
```

I assumed this was exported by '@storybook/react' but it shouldn't be necessary to assume it.

I also had troubles to make the decorator work. The fix of @jlpospisil worked thankfully!
